src/openrct2/paint/support/MetalSupports.cpp

```diff
@@ -384,14 +384,14 @@ bool MetalASupportsPaintSetup(
     }
 
     CoordsXYZ boundBoxOffset = CoordsXYZ(SupportBoundBoxes[segment], height);
-    si = height + special;
+    const auto si2 = height + special;
```

Comment:
`si` seems to be a register name (`esi`'s shorter sibling). Can we come up with something better?

Reply:
I cannot think of anything less wooden than "originalHeightPlusSpecialHeight", and I can’t say I’m happy with that.

Reply:
`combinedHeight` might be less specific, but more succinct?